x/ibc-hooks/wasm_hook.go

```diff
@@ -119,7 +131,7 @@ func isIcs20Packet(packet channeltypes.Packet) (isIcs20 bool, ics20data transfer
 	return true, data
 }
 
-func isMemoWasmRouted(memo string) (isWasmRouted bool, metadata map[string]interface{}) {
+func isMemoWasmRouted(memo, key string) (isWasmRouted bool, metadata map[string]interface{}) {
```

Comment:
Ohh its generalizing beyond wasm routing?

Perhaps need to update fn name?

Reply:
yeah, it's so we can reuse that function for callbacks. I didn't change the name because callbacks are also "wasm", but I'll add a gdoc and consider a better name

Reply:
renamed.